script/deploy/healthCheck.mts

```diff
@@ -356,6 +310,26 @@ const getOwnablContract = (address: Address, client: PublicClient) => {
   })
 }
 
+const checkOwnership = async (
+  name: string,
+  expectedOwner: Address,
+  deployedContracts: Record<string, Address>,
+  publicClient: PublicClient
+) => {
+  if (deployedContracts[name]) {
```

Comment:
I think I raised this before, not sure if you saw my comment. 
I feel that just reading from the deploy log can be dangerous. If a deployment goes wrong for some reason but the log is written anyway then we will not find this out.
I feel this script here should at least make sure that code is deployed at each contract address.

Reply:
Yes I plan to update this in a later PR